 kodilla-stream/src/main/java/com/kodilla/stream/portfolio/TaskList.java

```diff
@@ -0,0 +1,52 @@
+package com.kodilla.stream.portfolio;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
+
+public final class TaskList {
+    private final List<Task> tasks = new LinkedList<>();
+    private final String name;
+
+    public TaskList(String name) {
+        this.name = name;
+    }
+
+    public void addTask(Task task) {
+        tasks.add(task);
+    }
+
+    public boolean removeTask(Task task) {
+        return tasks.remove(task);
+    }
+
+    public List<Task> getTasks() {
+        return new LinkedList<>(tasks);
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public String toString() {
+        return "TaskList{" +
+                "name='" + name + '\'' + ",\n" +
+                "tasks=\n" + tasks + "\n" +
+                '}' + "\n";
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof TaskList)) return false;
+        TaskList taskList = (TaskList) o;
+        return name.equals(taskList.name);
```

Comment:
napisalem, ze to niepotrzebne. tym bardziej, ze twoja metoda equals zle dziala. dlaczego?